ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -400,6 +431,8 @@ export default {
       id: 'E12S Promise Chiseled Sculpture',
       netRegex: NetRegexes.headMarker({}),
       condition: (data, matches) => data.isDoorBoss && matches.target === data.me,
+      // Show headmarker number for whole phase
```

Comment:
Can this persisting number call say something like `1 (Inner)` or `4 (Outer)` instead?

Reply:
I'm not sure exactly how to handle this given that the tether and the headmarker are two separate triggers.

Can we rely on the order of the network lines for this? Here's an example from logs from last night:

```
27|2021-02-07T19:05:12.8460000-05:00|********|Player One|0000|0000|018D|0000|0000|0000||82513ce6107aeb3d25f6a8d10425e083
35|2021-02-07T19:05:12.8460000-05:00|********|Player One|40018B3B|Chiseled Sculpture|9598|0000|0011|40018B3B|000F|0000||be10e5a16e567a8c8273a7c224d44e74
27|2021-02-07T19:05:12.8460000-05:00|********|Player Two|0000|0000|0191|0000|0000|0000||fa22088288495908ac9be5d2af60c15f
35|2021-02-07T19:05:12.8460000-05:00|********|Player Two|40018B3A|Chiseled Sculpture|9598|0000|0011|40018B3A|000F|0000||049e70d56ab785dcbf7465666766d31e
27|2021-02-07T19:05:12.8460000-05:00|********|Player Three|0000|0000|018E|0000|0000|0000||1f86664445717844cc485cfa2a85c7b4
35|2021-02-07T19:05:12.8460000-05:00|********|Player Three|40018B3B|Chiseled Sculpture|9598|0000|0011|40018B3B|000F|0000||ec7f664a7167ac7fda13756541ec29c3
27|2021-02-07T19:05:12.8460000-05:00|********|Player Four|0000|0000|0192|0000|0000|0000||1206109c2010dce5a3c1f1fbb9da38e6
35|2021-02-07T19:05:12.8460000-05:00|********|Player Four|40018B3A|Chiseled Sculpture|9598|0000|0011|40018B3A|000F|0000||34c06f84a7da705714c452d678848847
27|2021-02-07T19:05:12.8460000-05:00|********|Player Five|0000|0000|018F|0000|0000|0000||056bab79dc151cac7891e05d02d52be2
35|2021-02-07T19:05:12.8460000-05:00|********|Player Five|40018B3B|Chiseled Sculpture|9598|0000|0011|40018B3B|000F|0000||acec9ff58f91b8b3452cc2f1fe65d310
27|2021-02-07T19:05:12.8460000-05:00|********|Player Six|0000|0000|0193|0000|0000|0000||52e93b2c86d3a83bf17d5341ff0b24da
35|2021-02-07T19:05:12.8460000-05:00|********|Player Six|40018B3A|Chiseled Sculpture|9598|0000|0011|40018B3A|000F|0000||a87da7109863367c4f7ffbe2749a7499
27|2021-02-07T19:05:12.8460000-05:00|********|Player Seven|0000|0000|0190|0000|0000|0000||5cb09d0ad1e4880c00cd02e55bb288d7
35|2021-02-07T19:05:12.8460000-05:00|********|Player Seven|40018B3B|Chiseled Sculpture|9598|0000|0011|40018B3B|000F|0000||d859bb36e9631dec384d97b77cf5436e
27|2021-02-07T19:05:12.8460000-05:00|********|Player Eight|0000|0000|0194|0000|0000|0000||372050caf1b8b86521e080785dce26bc
35|2021-02-07T19:05:12.8460000-05:00|********|Player Eight|40018B3A|Chiseled Sculpture|9598|0000|0011|40018B3A|000F|0000||2f430ac45f2df6311b6f6bdf3719614a
```

Reply:
I think you should be able to rely on the ordering.  The other option is to add a slight 0.25-0.5s delay from the number and then you can have text like `#1` if the tether wasn't there vs more information if it was.  That might be more robust for the future.